### ISSUE TITLE
dyld: Library not loaded:

### ISSUE THREAD

**Issue by DikeyKing** [mapbox/mapbox-gl-native-ios#196]
podfile:

```
def demo_pods  
    use_modular_headers!      
    pod 'SSZipArchive'    
    pod 'Mapbox-iOS-SDK',:modular_headers => true  
end   

target 'LightSDKDemo' do  
  use_modular_headers!  
  project '../LightSDKOrigin/LightSDKDemo.xcodeproj'  
  demo_pods  
  sdk_pods  
end  
```

error:

```
dyld: Library not loaded: @rpath/MapboxMobileEvents.framework/MapboxMobileEvents
  Referenced from: /private/var/containers/Bundle/Application/372C643A-3739-4B9C-BDAB-B6D4B36E3C7A/LightSDKDemo.app/Frameworks/Mapbox.framework/Mapbox
  Reason: image not found
Message from debugger: Terminated due to signal 6
```

**Mapbox SDK versions:** 5.7.0
**iOS/macOS versions:** 13.1.2/13.3
**Device/simulator models:** iPad Pro 12.9 / XR
**Xcode version:** Version 11.3.1 (11C504)

Is it related to apple's restriction?

**Comment by 1ec5** [comment]
@julianrex can you confirm if this is related to #157?

**Comment by DikeyKing** [comment]
```
020-03-09 17:10:12.508776+0800 LightSDKDemo[335:11444] [DYMTLInitPlatform] platform initialization successful
Total pre-main time: 371.41 milliseconds (100.0%)
         dylib loading time: 191.04 milliseconds (51.4%)
        rebase/binding time:  27.85 milliseconds (7.5%)
            ObjC setup time:  22.33 milliseconds (6.0%)
           initializer time: 129.98 milliseconds (34.9%)
           slowest intializers :
             libSystem.B.dylib :   2.89 milliseconds (0.7%)
    libMainThreadChecker.dylib :  25.35 milliseconds (6.8%)
          libglInterpose.dylib :  33.95 milliseconds (9.1%)
         libMTLInterpose.dylib :  13.81 milliseconds (3.7%)
                        Mapbox :  16.48 milliseconds (4.4%)
                  LightSDKDemo :  41.88 milliseconds (11.2%)
```

Adding Mapbox manually works, but still I'd like to integrate mapbox with cocoapods. 

I don't know if it's a apple's bug or weather cocoapods's

**Comment by julianrex** [comment]
You shouldn't need to add Mapbox manually with Cocoapods.

Can you confirm that you have a paid Apple developer account?

**Comment by DikeyKing** [comment]
> You shouldn't need to add MapboxMobileEvents manually with Cocoapods.
> 
> Can you confirm that you have a paid Apple developer account?

Yeah, it's a paid account.

**Comment by iNeedHelpX** [comment]
hey I had a similar issue. 

> dyld: Library not loaded: @rpath/MapboxMobileEvents.framework/MapboxMobileEvents
  Referenced from: /private/var/containers/Bundle/Application/372C643A-3739-4B9C-BDAB-B6D4B36E3C7A/LightSDKDemo.app/Frameworks/Mapbox.framework/Mapbox
  Reason: image not found

**Comment by rtxsc** [comment]
I've manage to solved problem #157 
i came across this issue when my **iPad Pro 10.5** was still at 13.3.1. After trying out suggestion by [iNeedHelpX](https://github.com/mapbox/mapbox-gl-native-ios/issues/196#issuecomment-605658440) it was unsuccessful. Then I just **updated** my iPad to the latest 13.4.1 firmware and that's it! Problem solved.

**macOS Catalina 10.15.3**
**iOS Developer University Program**  (Personal Team) **Role** User
**CocoaPods Mapbox-iOS-SDK**: 5.8.0 : use_frameworks!
**iOS 13.3.1** = Failed
**iOS 13.4.1** = Successful
**Xcode Version 11.4** (11E146)

❤️ from Malaysia. Stay stay stay home. COVID-19 😷😢

**Comment by julianrex** [comment]
Marking as a known issue, and solved by upgrading to 13.4.1
